api/src/utils/mailer-templates.ts

```diff
@@ -0,0 +1,169 @@
+import { IUser, IPitch, IPitchAggregate } from 'ssw-common';
+import User from '../models/user';
+
+export interface EmailMessage {
+  to: string;
+  from: string;
+  subject: string;
+  html: string;
+}
+type userInfo = { name: string; email: string };
+type teamsUsers = Record<string, userInfo[]>;
+
+const getAllTeams = (teamNamesToUsers: teamsUsers): string => {
+  const message: string[] = [];
+  for (const key in teamNamesToUsers) {
+    message.push(`<b>${key}</b> <br>`);
+    for (const member of teamNamesToUsers[key]) {
+      message.push(`<li> ${member.name} - ${member.email} </li>`);
+    }
+  }
+  return message.join('');
+};
+
+export const declinedMessage = (
+  author: IUser,
+  pitch: IPitch,
+  admin: IUser,
+): EmailMessage => ({
+  //TODO: find out what those links should point to
+  to: author.email,
+  from: process.env.EMAIL_USERNAME,
+  subject: `Pitch "${pitch.title}" Declined`,
+  html: `<div>
+                <dt>Hi ${author.preferredName || author.firstName},</dt>
+                <br>
+                <dt>Thank you for submitting your pitch <a href='"${
+                  pitch.assignmentGoogleDocLink
+                }"'>"${
+    pitch.title
+  }"</a> Unfortunately, your pitch was declined. </dt>
+                If you have any questions or need any additional support, please contact ${
+                  admin.email
+                }. 
+                Check out more <a href='www.google.com'>pitch-writing resources</a>. In the meantime, feel free to check the 
+                  <a href='http://south-side-weekly.vercel.app/pitches'>pitch doc</a> for potential new stories to claim!</dt>
+                <br>
+                <dt>Thank you,
+                    <br>
+                ${admin.preferredName || admin.firstName}
+                </dt>
+                </div>`,
+});
+
+export const approvedMessage = (
```

Comment:
Your other functions say approveClaim/rejectedClaim, could this get chagned to approvePitch/declinePitch